### PR TITLE
Add CSRF protection to the poste send-messages action

### PR DESCRIPTION
**Pull request by philbogle**
Add CSRF protection to the poste send-messages action using the `rack_csrf` gem.

While we eventually plan to migrate many Sinatra routes to Rails, this will take a while and it's valuable to have a way to have some CSRF protection in Pegasus in the meanwhile.

Currently the CSRF protection is restricted to only that action, since modifications to the forms and/or AJAX request logic are required to include CSRF token.  A similar approach will be used in future PRs for other actions that we want to protect.

### CODE REVIEW

**Comment by philbogle** [comment]
@wjordan @laurelfan FYI  

Let me know if the approach here looks good to you.

File: pegasus/config.ru [hunk 1]
```diff
@@ -1,5 +1,9 @@
 require File.expand_path('../router', __FILE__)
 
+require 'rack/csrf'
+use Rack::Session::Cookie, secret: (CDO.sinatra_session_secret || 'dev_mode')
```
Comment (aoby) [inline pegasus/config.ru:4]: Where is CDO.sinatra_session_secret set? Is this the right property name?

**Comment by aoby** [comment]
circleCI failed. It looks unrelated, but re-run that first.

lgtm aside from the above

File: pegasus/config.ru (hunk 1 above)
Comment (philbogle) [inline pegasus/config.ru:4]: This is a new property I have defined in the production envivronment.  You can see it using "knife environment edit production".

**Pull request merged**